src/XRP/shared/rippled-web-socket-schema.ts

```diff
@@ -0,0 +1,111 @@
+/* Schema for adding type information to Web Socket objects. */
+
+/**
+ * The standard format for a request to the JSON RPC exposed by a rippled node.
+ * @see https://xrpl.org/request-formatting.html
+ */
+interface WebSocketRequestOptions {
+  command: string
```

Comment:
it might be nice to make an enum for the different commands: https://xrpl.org/public-rippled-methods.html. you could also use it in `subscribeToAccount`

Reply:
Should I dump the whole list there now? I created the enum for now, but it's only got the one method atm